internals/webpack/webpack.base.js

```diff
@@ -96,5 +100,4 @@ module.exports = (options) => ({
   devtool: options.devtool,
   target: 'web', // Make web variables accessible to webpack, e.g. window
   stats: false, // Don't show stats in the console
-  progress: true,
```

Comment:
Woah that's weird. @TheLarkinn why was this removed, or is this a bug in the validator? /cc @gajus